recipes/ml_insights/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "ml_insights" %}
```

Comment:
I think we generally try to use the same name as on pypi, so this should be named `ml-insights`, but I can't find the official documentation/recommendation. @conda-forge/staged-recipes, can anyone remind me if this is correct?

Reply:
in any case I've changed the name in the recipe and the folder name to ml-insights as suggested

Reply:
@synapticarbors is correct, we try to use the same name. However, pip handles both _ and - as "the name" when installing. To mimick that we can use multiple outputs. Take a look at the pre-commit recipe if you want to go that route.